assets/scss/styles.scss

```diff
@@ -13,6 +13,7 @@
  */
 
 html {
+  font-size: initial !important;
```

Comment:
this is only needed if we end up using flatpickr. Otherwise it might now be needed. Maybe it is better to revert that for now.

Reply:
Removed for now, until we add flatpickr, if needed.